src/utils/registry.ts

```diff
@@ -660,6 +660,18 @@ function markerFilePath(browserDirectory: string): string {
   return path.join(browserDirectory, 'INSTALLATION_COMPLETE');
 }
 
+export function buildPlaywrightCLICommand(cliLanguage: string, parameters: string): string {
+  const cliLanguageToCLIBuilder: Partial<Record<string, (parameters: string) => string>> = {
```

Comment:
Why Partial?

Reply:
This ends up in a `string` type otherwise. I rewrote this part, so not needed anymore.